channeldb/invoices.go

```diff
@@ -172,6 +172,9 @@ type Invoice struct {
 	// for this invoice can be stored.
 	PaymentRequest []byte
 
+	// TODO(joostjager): Store invoice cltv as separate field in database so
+	// that it doesn't need to be decoded from the payment request.
```

Comment:
I still haven't decided whether grouping migrations is good or not good. Good to have a single version, but not good because it may be more difficult to find the problem if there is a problem.

Migration for cltv delta added.